examples/job/ml-train-model-predict-properties.ipynb

```
@@ -139,13 +133,13 @@
    "outputs": [],
    "source": [
     "# Import settings file and utils file\n",
-    "from utils.settings import ENDPOINT_ARGS, ACCOUNT_ID\n",
-    "from utils.api import (\n",
+    "from mat3ra.notebooks.utils import ENDPOINT_ARGS, ACCOUNT_ID\n",
+    "from mat3ra.notebooks.utils.other.api import (\n",
```

Ruff (0.15.12)

[error] `await` statement outside of a function

(F704)